projects/dotcms-field-elements/src/components/dot-select/dot-select.e2e.ts

```diff
@@ -1,93 +1,276 @@
+                const htmlElement = await getSelect(page);
+                expect(await htmlElement.getAttribute('disabled')).toBe('');
+            });
         });
 
-        it('should be invalid, touched & dirty and the error msg should display', async () => {
-            await page.select('select', '');
-            await page.waitForChanges();
-            expect(element).toHaveClasses(['dot-invalid', 'dot-touched', 'dot-dirty']);
+        describe('name', () => {
+            const value = 'test';
+
+            it('should render attribute in label and select', async () => {
+                element.setProperty('name', value);
+                await page.waitForChanges();
+
+                const selectElement = await getSelect(page);
+                const idValue = await selectElement.getProperty('id');
+                expect(idValue.indexOf(value)).toBeGreaterThan(-1);
+
+                const labelElement = await getDotLabel(page);
+                expect(await labelElement.getProperty('name')).toBe(value);
+            });
+
+            it('should not render attribute in label and select', async () => {
+                const selectElement = await getSelect(page);
+                const idValue = await selectElement.getProperty('id');
+                expect(idValue.indexOf(value)).toBe(-1);
+
+                const labelElement = await getDotLabel(page);
+                expect(await labelElement.getProperty('name')).toBe('');
+            });
+
+            it('should not break with invalid data', async () => {
+                const wrongValue = { a: 1 };
+                element.setProperty('name', wrongValue);
+                await page.waitForChanges();
+
+                const selectElement = await getSelect(page);
+                const idValue = await selectElement.getProperty('id');
+                expect(idValue.indexOf(wrongValue)).toBe(-1);
+            });
         });
 
-        it('it should set options blank when an Object is passed as options', async () => {
-            element.setProperty('options', { noValid: true });
-            await page.waitForChanges();
-            const options = await page.find('option');
-            expect(options).toBe(null);
+        describe('label', () => {
+            it('should render attribute in label', async () => {
+                const value = 'test';
+                element.setProperty('label', value);
+                await page.waitForChanges();
+
+                const labelElement = await getDotLabel(page);
+                expect(await labelElement.getProperty('label')).toBe(value);
+            });
+
+            it('should not break with invalid data', async () => {
+                const wrongValue = { a: 1 };
+                element.setProperty('label', wrongValue);
+                await page.waitForChanges();
+
+                const labelElement = await getDotLabel(page);
+                expect(labelElement).toBeTruthy();
```

Comment:
this is testing that `<dot-label>` exist, should not check for the property value itself ?